ironfish-cli/src/commands/faucet.ts

```diff
@@ -41,7 +41,7 @@ export class FaucetCommand extends IronfishCommand {
 
     if (!email) {
       email = (await CliUx.ux.prompt('Enter your email to stay updated with Iron Fish', {
-        required: false,
+        required: true,
```

Comment:
Email isn't required, as it uses your public key if you don't specify an email.

Reply:
Oh oops, you're right. I thought it was required cause leaving email blank results in `Collecting your funds... You entered an invalid email.`. Turns out it submits an empty string to the API, which always fails. So we need to pass `undefined` if a user skips email.